.browserslistrc

```diff
@@ -8,5 +8,3 @@ last 2 versions
 # ensure these are supported:
 Firefox ESR
 IE 11
-# because no one likes a dead browser:
-not dead
```

Comment:
this is breaking on develop. i guess its a browserslist 3.0 thing, which some of our deps don't support yet (specifically cssnano in css-loader).